Lib/test/test_bytes.py

```diff
@@ -1599,6 +1599,8 @@ def test_extend(self):
         a = bytearray(b'')
         a.extend([Indexable(ord('a'))])
         self.assertEqual(a, b'a')
+        a = bytearray(b'abc')
+        self.assertRaises(TypeError, a.extend, 'def', msg="Expected iterable of integers; got: 'str'")
```

Comment:
`msg` args are for display when a test fails, not part of the test*
```suggestion
        self.assertRaises(TypeError,
                          "Expected iterable of integers; got: 'str'",
                          a.extend, 'def')
```

Reply:
needs more

Reply:
Oops. Thanks for catching that. I think I need to change this in another PR as well.